source/dataman/AMNumber.h

```diff
@@ -128,7 +128,7 @@ class AMNumber
 	/// Print as a string
 	QString toString(QChar format = 'g', int precision = 19);
 
-	QString toString(int precision);
+	const QString toString(int precision);
```

Comment:
Sorry for the confusion, but this should be a const function, not a function that returns a const variable.  It should be:

```c++
QString toString() const;
```

not

```c++
const QString toString();
```
If you're confused about the usage of the various const's, feel free to come by.